fuzzing/test_case_assertion_provider.go

```diff
@@ -234,3 +235,79 @@ func (t *AssertionTestCaseProvider) callSequencePostCallTest(worker *FuzzerWorke
 
 	return shrinkRequests, nil
 }
+
+// encounteredAssertionFailure takes in a panic code and a config.AssertionModesConfig and will determine whether the
+// panic code that was hit should be treated as a failing case - which will be determined by whether that panic
+// code was enabled in the config. Note that the panic codes are defined in the abiutils package and that this function
+// panic if it is provided a panic code that is not defined in the abiutils package.
+// TODO: Also note that this is a terrible design and if there is a better way to maintain assertion and panic logic, it
```

Comment:
Comment spacing fix here

Reply:
Resolved in [cd27dcb](https://github.com/crytic/medusa/pull/161/commits/cd27dcb384e4f0ae2dc52f67534a6b329b1c1ec1)